### PR TITLE
Fix adversarial attacks

### PR DESCRIPTION
**Pull request by LaurentMnr95**
## Types of changes

- [] Docs change / refactoring / dependency upgrade
- [x] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [] The documentation is up-to-date with the changes I made.
- [] I have read the [**CONTRIBUTING**](https://facebookresearch.github.io/nevergrad/contributing.html) document and completed the CLA (see [**CLA**](https://facebookresearch.github.io/nevergrad/contributing.html#contributor-license-agreement-cla)).
- [] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

File: nevergrad/functions/images/dataset_folder.py [hunk 1]
```diff
@@ -0,0 +1,222 @@
+from torchvision.datasets.vision import VisionDataset
```
Comment (jrapin) [inline nevergrad/functions/images/dataset_folder.py:1]: please use pep8 compatible naming for modules (aka datasetfolder.py, no `_`)

File: nevergrad/functions/images/dataset_folder.py [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+from PIL import Image
+import time
+import os
+import os.path
+import sys
+import pickle
```
Comment (jrapin) [inline nevergrad/functions/images/dataset_folder.py:7]: pylint will complain that standard imports should come first

File: nevergrad/functions/images/dataset_folder.py [hunk 3]
```diff
@@ -0,0 +1,222 @@
+from torchvision.datasets.vision import VisionDataset
+from PIL import Image
+import time
+import os
+import os.path
```
Comment (jrapin) [inline nevergrad/functions/images/dataset_folder.py:5]: I would advice to always use pathlib instead, it makes *everything* clearer

File: nevergrad/functions/images/dataset_folder.py (hunk 1 above)
Comment (jrapin) [reply nevergrad/functions/images/dataset_folder.py:1]: This is missing the standard headers

**Review by teytaud — APPROVED** [review]

**Comment by teytaud** [comment]
Before merging: run at least once + check if the fix in 994 is in it.

File: nevergrad/functions/images/core.py [hunk 4]
```diff
@@ -246,19 +248,22 @@ def make_folder_functions(
...
+
+        transform = tr.Compose([tr.Resize(256), tr.CenterCrop(224), tr.ToTensor()])
         if folder is None:
             x = torch.zeros(1, 3, 224, 224)
             _, pred = torch.max(classifier(x), axis=1)
             data_loader: tp.Iterable[tp.Tuple[tp.Any, tp.Any]] = [(x, pred)]
         elif Path(folder).is_dir():
-            ifolder = torchvision.datasets.ImageFolder(folder, transform)
-            data_loader = torch.utils.DataLoader(
+            print(f"Folder  = {folder}")
```
Comment (jrapin) [inline nevergrad/functions/images/core.py:259]: is this needed? using a logger would be better but if that helps for now lets keep in

**Review by jrapin — APPROVED** [review]
if everything works in the tests then good ;)

File: mypy.ini [hunk 5]
```diff
@@ -1,5 +1,6 @@
 [mypy]
 
+
```
Comment (jrapin) [inline mypy.ini:3]:
```suggestion
```

**Comment by jrapin** [comment]
Replaced by #1257

**Pull request closed without merging**